ui/app/AppMain.qml

```diff
@@ -592,13 +593,22 @@ Item {
         }
 
         Connections {
-            target: profileModel.contacts
+            target: profileModel
+            ignoreUnknownSignals: true
+            enabled: removeMnemonicAfterLogin
+            onInitialized: {
+                mnemonicModule.remove()
+            }
+        }
```

Comment:
Surprised to see this change in this commit. Is this still related to contacts refactor?